helm/charts/services/amplication-client/values.yaml

```diff
@@ -34,5 +34,5 @@ config:
     NX_REACT_APP_SERVER_URI: https://server.amplication-dev.com
     NX_REACT_APP_DATA_SOURCE: https://server.amplication-dev.com/graphql
     NX_REACT_APP_GITHUB_CONTROLLER_LOGIN_URL: https://server.amplication-dev.com/github
-    NX_REACT_APP_GITHUB_CLIENT_ID: a4e9055e5f21448d6de7
+    NX_REACT_APP_GITHUB_AUTH_ENABLED: a4e9055e5f21448d6de7
```

Comment:
That looks weird, env with enabled at the end usually means boolean value, here we have string. What was the intention?

Reply:
Great point. 
I'll update the values accordingly